newsfragments/336.bugfix.rst

```diff
@@ -0,0 +1 @@
+Restore symbols missing after compilers move: `distutils.ccompiler.compiler_class` and `distutils._msvccompiler.PLAT_SPEC_TO_RUNTIME` -- by :user:`Avasam`
```

Comment:
Let's rename this to 4876.bugfix.rst, since it'll get manifest in Setuptools. Thanks.

Reply:
Right, forgot about that.